src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1640,13 +1357,15 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
             // Given a board where user has resign
             await prepareTestUtilsFor(UserMocks.CREATOR_AUTH_USER);
             await doMove(FIRST_MOVE, true);
-            await receiveNewMoves([FIRST_MOVE_ENCODED, SECOND_MOVE_ENCODED], 2, 1799999, 1800 * 1000);
+            await receiveNewMoves(1, [SECOND_MOVE_ENCODED]);
             await receivePartDAOUpdate({
                 winner: UserMocks.CREATOR_MINIMAL_USER,
                 loser: UserMocks.OPPONENT_MINIMAL_USER,
                 result: MGPResult.RESIGN.value,
                 request: null,
-            }, 3);
+            });
+            await partService.addAction('configRoomId', Player.ONE, 'EndGame');
```

Comment:
pareil là et sans doute d'autres endroit, c'p-e plus clair de dire "quand on reçois" différemment dans les tests, même si ça passe par le "on envoi" (tu vois ?)
Genre "receivePartDAOUpdate" plus haut fait ça quoi

Reply:
ok je repasse sur les tests en ajoutant un `receiveAction` pour être explicite !

Reply:
ok, next step: explain why it's allowed by test to add action as the opponent ! :D

Reply:
car on est pas dans le PartDAO donc les règles de sécu sont pas enforcées !